backend/src/models/user.go

```diff
@@ -77,6 +76,11 @@ type UpdateUserRequestBody struct {
 	Year      Year    `json:"year" validate:"omitempty,min=1,max=6"`
 }
 
+type LoginUserResponseBody struct {
+	Email    string `json:"email" validate:"required,email"`
```

Comment:
```suggestion
	Email    mail.Address `json:"email" validate:"required,email"`
```
similar idea to below, should we use the [stdlib mail package](https://pkg.go.dev/net/mail#Address)? if we do, do we even need to validate email 🤔